web/app/js/components/ServiceMesh.jsx

```diff
@@ -27,6 +25,58 @@ const serviceMeshDetailsColumns = [
     className: "numeric"
   }
 ];
+
+const barColor = percentMeshed => {
+  if (percentMeshed <= 0) {
+    return "neutral";
+  } else {
+    return "good";
+  }
+};
+
+const namespacesColumns = ConduitLink => [
+  {
+    title: "Namespace",
+    dataIndex: "namespace",
+    key: "namespace",
+    defaultSortOrder: "ascend",
+    sorter: (a, b) => (a.namespace || "").localeCompare(b.namespace),
+    render: d => <ConduitLink to={"/namespaces?ns=" + d}>{d}</ConduitLink>
+  },
+  {
+    title: "Meshed pods",
+    dataIndex: "meshedPodsStr",
+    key: "meshedPodsStr",
+    className: "numeric",
+    sorter: (a, b) => numericSort(a.totalPods, b.totalPods),
+  },
+  {
+    title: "Mesh completion",
+    key: "meshification",
+    sorter: (a, b) => numericSort(a.meshedPercent.get(), b.meshedPercent.get()),
+    render: row => {
+      let containerWidth = 132;
+      let percent = row.meshedPercent.get();
+      let barWidth = percent < 0 ? 0 : Math.round(percent * containerWidth);
```

Comment:
how does `percent < 0` happen?

Reply:
Percentage.js returns -1 if the denominator is 0:

```
export default function Percentage(numerator, denominator) {
  this.decimal = -1;
  if (denominator !== 0) {
    this.decimal = numerator / denominator;
  }
}
```